packages/teams-js/src/public/app/app.ts

```diff
@@ -297,6 +305,12 @@ export interface PageInfo {
    */
   frameContext: FrameContexts;
 
+  /**
+   * The mode or surface where the page is rendered (e.g. sidePanel, meetingStage, etc.)
+   * This will be used by the app developers in future to know where in the host the app is rendered instead of the frameContext.
```

Comment:
Has this not been implemented on host SDK yet? Are we going to have to change this documentation when it is available?

Reply:
Yeah we have to change the description once we enumarate all the rendering surfaces from all the hosts. especially the logic around the mapping for frameContext to rendering Surface.